### PR TITLE
Adds note and updates Clair disconnected information

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/clair-disconnected.adoc [hunk 1]
```diff
@@ -1,50 +1,67 @@
...
-```
+----
 
+[id="mapping-repositories-to-cpe-information"]
+== Mapping repositories to Common Product Enumeration (CPE) information
 
+Clair's {rhel} scanner relies on a Common Product Enumeration (CPE) file to properly map RPM packages to the corresponding security data to produce matching results. The CPE file must be present, or access to the file must be allowed, for the scanner to properly process RPM packages. If the file is not present, RPM packages installed in the container image will not be scanned. 
 
-== Mapping repositories to Common Product Enumeration (CPE) information
+* The `repos2cep` JSON mapping file is published at link:https://www.redhat.com/security/data/metrics/repository-to-cpe.json[Red Hat Repository-to-CPE JSON].
```
Comment (crozzy) [inline modules/clair-disconnected.adoc:53]: repos2cpe

File: modules/clair-disconnected.adoc [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
+Clair uses a set of components called `Updaters` to handle the fetching and parsing of data from various vulnerability databases. `Updaters` are set up by default to pull vulnerability data directly from the internet and work for immediate use. For customers in disconnected environments without direct access to the internet, this poses a problem. Clair supports these environments by working with different types of update workflows that take into account network isolation. Using the `clairctl` command line utility, any process can easily fetch `Updater` data from the internet by using an open host, securely transfer the data to an isolated host, and then import the `Updater` data on the isolated host into Clair itself. 
 
-The steps are as follows.
+[NOTE]
+====
+Clair enrichment data is currently unsupported in disconnected environments.
```
Comment (crozzy) [inline modules/clair-disconnected.adoc:9]: At the moment the only enrichment data is CVSS data, would it be possible to mention CVSS as well for people who'd be looking for that?

Comment (stevsmit) [reply modules/clair-disconnected.adoc:9]: @crozzy Do you want it put here, or in the section above? https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/manage_red_hat_quay/clair-intro2#clair-cve

If you want it here, I might say... 

"Currently, Clair enrichment data is CVSS data. Enrichment data is currently unsupported in disconnected environments. "

If I in the other section, I could maybe say... 

"With Clair v4.2,  CVSS enrichment data is now viewable in the Quay UI."

WDYT?

Comment (crozzy) [reply modules/clair-disconnected.adoc:9]: Yeah, I think that makes sense and will be best for users searching either "cvss" or "enrichment"

**Pull request merged**